test/scenarios/periodics/cos9-src@flannel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-centos.ks.template ""
+    launch_vm --boot_blueprint centos9
```

Comment:
Let's convert it to a bootc test
```suggestion
    launch_vm --boot_blueprint centos9-bootc --bootc
```

Reply:
Done.